.github/workflows/docker-image.yml

```diff
@@ -32,15 +32,16 @@ jobs:
       with:
         username: ${{ github.actor }}
         password: ${{ secrets.DOCKER_TOKEN }}
-        image_name: ${{ github.repository_owner }}dev/ledgersmb_circleci-${{ matrix.browser }}
-        image_tag: latest
-        push_git_tag: ${{ github.event_name == 'push' || github.event.pull_request.merged }}
+        image_name: ${{ github.repository_owner }}dev/ledgersmb_circleci-browsers
+        image_tag: ${{ matrix.browser }}
+        push_git_tag: ${{ github.event.pull_request.merged }}
         context: ${{ matrix.browser }}
-        push_image_and_stages: ${{ github.event_name == 'push' || github.event.pull_request.merged }}
+        stages_image_name: "${{ github.repository_owner }}dev/ledgersmb_circleci-browsers-${{ matrix.browser }}-stages"
+        push_image_and_stages: false
   perl:
     strategy:
       matrix:
-        version: ['5.28', '5.30', '5.32', '5.34']
+        version: ['5.32', '5.34']
```

Comment:
@ylavoie Just asking: don't we need these older Perl versions in order to test the 1.7-1.9 branches?

Reply:
Kept